### PR TITLE
Estute/te 2618

### PR DESCRIPTION
**Pull request by estute**
This will install and configure the email ext plugin. A fair amount of the plugin configuration can be pulled from the mailer plugin, which we already install, as long as this script is run after the mailer plugin is installed.

### CODE REVIEW

File: src/main/groovy/5configureEmailExtension.groovy [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    'StillFailingTrigger',
+    'StillUnstableTrigger',
+    'SuccessTrigger',
+    'UnstableTrigger',
+    'XNthFailureTrigger'
+]
+// verify that only valid trigger classes are used to set default triggers
+emailConfig.TRIGGERS.each { trigger ->
+    if (! triggerList.contains(trigger)) {
+        logger.severe('NOPE')
```
Comment (jmbowman) [inline src/main/groovy/5configureEmailExtension.groovy]: Maybe a slightly more verbose error message would be helpful here? 😃

Comment (estute) [reply src/main/groovy/5configureEmailExtension.groovy]: Ha. Oops, sorry forgot to make it nice for other people

File: e2e/test_email_ext_configuration.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    def setUp(self):
+        super(TestEmailExtConfiguration, self).setUp()
+        config_path = os.getenv('CONFIG_PATH')
+        try:
+            yaml_contents = open(
+                    "{}/email_ext_config.yml".format(config_path), 'r'
+                    ).read()
+        except IOError:
+            pass
+        self.email_config = yaml.load(yaml_contents)
```
[the remark this answers is not in the document]
Comment (estute) [reply e2e/test_email_ext_configuration.py]: I'm not too concerned about loading arbitrary yaml files as we will only be running it on internally drafted configs, but it's an easy enough fix

File: e2e/test_email_ext_configuration.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+from bok_choy.web_app_test import WebAppTest
+from pages.email_ext_subpage import EmailExtConfigurationSubPage
+
+class TestEmailExtConfiguration(WebAppTest):
+    
+    def setUp(self):
+        super(TestEmailExtConfiguration, self).setUp()
+        config_path = os.getenv('CONFIG_PATH')
+        try:
+            yaml_contents = open(
```
[the remark this answers is not in the document]
Comment (estute) [reply e2e/test_email_ext_configuration.py]: If it is ok with you, I would like to just upgrade the tests to python 3 in one fell swoop, but in a separate PR. What do you think?

[the remark this answers is not in the document]
Comment (jmbowman) [reply e2e/test_email_ext_configuration.py]: Sure; just want to make sure we don't keep making the switch harder by writing more and more Python 2-only code.

**Pull request merged**